cli/app.go

```diff
@@ -26,8 +26,9 @@ const (
 	apiKeyCreateFlagOrgID = "org-id"
 	apiKeyCreateFlagName  = "name"
 
-	loginFlagKeyID = "key-id"
-	loginFlagKey   = "key"
+	loginBrowserFlag = "disable-browser-open"
```

Comment:
I'm definitely open to better names for this flag 😄

Reply:
Yeah can we name this `loginFlagDisableBrowser`? Generally, we're doing `[whichAction]Flag[WhatDoesFlagDo]`.